includes/Modules/Sign_In_With_Google.php

```diff
@@ -221,33 +392,54 @@ private function render_signin_button() {
 			return;
 		}
 
-		$redirect_url = add_query_arg( 'action', 'google_auth', wp_login_url() );
-		if ( substr( $redirect_url, 0, 5 ) !== 'https' ) {
+		$login_uri = add_query_arg( 'action', 'google_auth', wp_login_url() );
+		if ( substr( $login_uri, 0, 5 ) !== 'https' ) {
 			return;
 		}
 
+		$redirect_to = $this->context->input()->filter( INPUT_GET, 'redirect_to' );
+		$redirect_to = trim( $redirect_to );
+
 		// Render the Sign in with Google button and related inline styles.
 		?>
-<!-- <?php echo esc_html__( 'Sign in with Google button added by Site Kit', 'google-site-kit' ); ?> -->
+<!-- Sign in with Google button added by Site Kit -->
 <?php /* phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript */ ?>
 <script src="https://accounts.google.com/gsi/client"></script>
```

Comment:
Why not enqueue this? Also, it should be loaded async, no? https://developers.google.com/identity/gsi/web/guides/client-library

Reply:
> Also, it should be loaded async, no?

Not necessarily. It makes sense to load it async on the frontend where page speed matters, but for the login page it is less relevant.

> Why not enqueue this?

We can enqueue it but that will just add unnecessary complexity here. Do you really think we need to do it?

Reply:
SGTM